File: dataset_reduzido/codenet/Java/Code/s725559666.java
import java.util.Scanner;

class Main {

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int a = scan.nextInt();
		int b = scan.nextInt();
		int c = scan.nextInt();
		scan.close();
		if(a < b && b < c && a < c) {
			System.out.println("Yes");
			
		}else {
			System.out.println("No");
		}
		
		// TODO 自動生成されたメソッド・スタブ

	}

}

File: output/Qwen/codenet/Java/Python/s725559666.py
def main():
    scanner = Scanner(sys.stdin)
    a = scanner.nextInt()
    b = scanner.nextInt()
    c = scanner.nextInt()
    scanner.close()
    
    if a < b and b < c and a < c:
        print("Yes")
    else:
        print("No")

if __name__ == "__main__":
    main()